.ci/windows-steps.yaml

```diff
@@ -12,9 +12,14 @@ steps:
     nuget install boost -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_random-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_math_c99-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
-    nuget install OpenBLAS -o $(Agent.ToolsDirectory)
     nuget install unofficial-flayan-cereal -o $(Agent.ToolsDirectory)
-
+    nuget install ensmallen -o $(Agent.ToolsDirectory) -Version 2.17.0
+    ## Delete all ensmallen dependencies including armadillo headers, we do not need them here
```

Comment:
I'm not sure I get the reason why we download the ensmallen package from nuget, and delete everything besides the header files. In the linux build we just download the header files and copy everything into the target directory. Is there anything that I'm missing here?

Reply:
Frankly, I do not remember why we went into NuGet direction, there was a strong reason, but I can not remember why

Reply:
After all of this suffering, I think we can merge this pull request. Whatever the first reason was, it is better to use NuGet for future versions, we will have to change only the number now for ensmallen